numpy/core/src/multiarray/number.c

```diff
@@ -1128,7 +1139,7 @@ NPY_NO_EXPORT PyNumberMethods array_as_number = {
     (binaryfunc)array_divmod,                   /*nb_divmod*/
     (ternaryfunc)array_power,                   /*nb_power*/
     (unaryfunc)array_negative,                  /*nb_neg*/
-    (unaryfunc)_array_copy_nice,                /*nb_pos*/
```

Comment:
With some regret, I decided to undo this change, because of the potential for a backwards compatibility break (we're pretty paranoid about those).